checker/src/test/java/org/checkerframework/checker/test/junit/I18nTest.java

```diff
@@ -14,7 +14,11 @@ public class I18nTest extends CheckerFrameworkPerDirectoryTest {
      * @param testFiles the files containing test code, which will be type-checked
      */
     public I18nTest(List<File> testFiles) {
-        super(testFiles, org.checkerframework.checker.i18n.I18nChecker.class, "i18n");
+        super(
+                testFiles,
+                org.checkerframework.checker.i18n.I18nChecker.class,
+                "i18n",
+                "-AignoreTargetLocation");
```

Comment:
Also here, please add a comment why we ignore the target locations. Same for other uses in tests, unless you test the option somewhere.

Reply:
I have tested this `targetlocation` meta-annotation for h1h2checker and pico.

Ignoring it simply because too many errors in test cases if the enforcement is enabled, like the usage of bottom on parameters. I am not sure what i should comment.

Reply:
`checker/tests/lock/GuardSatisfiedTest.java`,  `checker/tests/nullness/KeyForLub.java` report `type.invalid.annotations.on.location` errors because related JUnit tests don't have that ignore option.